src/messaging/ReliableMessageMgr.cpp

```diff
@@ -24,6 +24,7 @@
 #include <errno.h>
 #include <inttypes.h>
 
+#include <app/util/af-types.h>
```

Comment:
what bits caused us to include this?

I am a bit concerned whenever we include any framework bits deeper in our stack. Layering wise, messaging should be lower level than app (which should be highest level) ... although maybe types are somewhat generic.

Reply:
I see `src/app/util/types_stub.h` contains UNUSED_VAR.

however I still disagree about including app/util. Lets put unused var at some other lower layer than app and not tie in to app/util. We can have a core header or support header instead.

Reply:
@andy31415 I can move the UNUSED_VAR(nodeId) declaration wherever you'd like :) Let me know.

I just want to use it ;) I see lots of the code base has just #defined their own in place

Reply:
I'm just going to move the macro back in, and I'll file an issue for folks to debate which layer to include it at.

Reply:
https://github.com/project-chip/connectedhomeip/issues/32113